client/verta/verta/deployment/resources.py

```diff
@@ -1,18 +1,39 @@
 # -*- coding: utf-8 -*-
 
 import abc
+import re
 
 from ..external import six
 
 
 @six.add_metaclass(abc.ABCMeta)
 class _Resource(object):
-    raise NotImplementedError
+    def __init__(self, parameter):
+        self.parameter = parameter
 
 
 class CpuMilli(_Resource):
-    raise NotImplementedError
+    def __init__(self, parameter):
+        try:
+            if parameter <= 0:
+                raise ValueError('Wrong parameter provided')
+        except TypeError:
+            raise ValueError('Wrong parameter type provided')
```

Comment:
Let's have a slightly more specific error message to help the user, for example:
https://github.com/VertaAI/modeldb/blob/ef91ff1/client/verta/verta/deployment/update/_strategies.py#L46-L50

Reply:
replaced